src/libguac/guacamole/user.h

```diff
@@ -493,6 +493,30 @@ guac_user* guac_user_alloc();
 void guac_user_free(guac_user* user);
 
 /**
+ * Handles the portion of a user's Guacamole protocol handshake following the
+ * "select" instruction all subsequent I/O, automatically populating the
```

Comment:
Should there be some more punctuation here or something? This first sentence is really hard to parse.